pom.xml

```diff
@@ -125,6 +139,12 @@
             <artifactId>graal-sdk</artifactId>
             <version>${graalvm.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.graalvm.truffle</groupId>
+            <artifactId>truffle-api</artifactId>
+            <version>${graalvm.version}</version>
+            <scope>runtime</scope>
```

Comment:
Runtime only dependency. Possibly redundant, as `org.graalvm.js:js` would probably bring this one in anyway.

Reply:
But is it true this `pom.xml` could be shorter: proposed in [PR-7](https://github.com/graalvm/graal-js-jdk11-maven-demo/pull/7/).